scripts/install.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+
+var log = require('npmlog');
+var execSync = require('child_process').execSync;
+
+// Default command
+var fetchCommand = 'node-pre-gyp install --fallback-to-build=false';
+var buildCommand = 'cmake-js compile';
+
+// ==== Determine install options ====
+// Skip the fetch stage. '--no-fetch', or '--fetch=false'
+var skipFetch = process.env.hasOwnProperty('npm_config_fetch') && !process.env['npm_config_fetch'];
```

Comment:
From what I observed and learned from other project (such as node-pre-gyp), NPM parses package.json and commandline arguments and generates some env variables with prefix npm_config_

However, actually I didn't find any info about this on npm documentation. The most related would be this:
[https://docs.npmjs.com/misc/config#environment-variables](https://docs.npmjs.com/misc/config#environment-variables)


---
In reply to: [133246502](https://github.com/Microsoft/napajs/pull/41#discussion_r133246502) [](ancestors = 133246502)